### ISSUE TITLE
An option to navigate in xProfile fields is not appearing on the front-end.

### ISSUE THREAD

**Issue by HardeepAsrani** [cuny-academic-commons/cbox-theme#213]
The following line on `cbox-theme/blob/master/members/single/profile/edit.php` file:

```
<?php if ( cbox_profile_has_multiple_tabs() ) : ?>
<ul class="button-nav">

<?php bp_profile_group_tabs(); ?>

</ul>
<?php endif; ?>
```

is the reason why there's no option to select xProfile field group in the front end. Removing the conditional php lines will solve this issue :)

**Comment by HardeepAsrani** [comment]
Just wanted to know if this got fixed so we could solve this issue. :)

**Comment by boonebgorges** [comment]
Yes, it should be fixed as of the most recent version of Commons In A Box. Please let us know if you're still having problems.

**Comment by HardeepAsrani** [comment]
Hi,

The guy who asked me to fix this first also contacted me to fix this issue again. He's using the latest version of BuddPress and Commons in a Box theme. :)

**Comment by boonebgorges** [comment]
I went with a slightly different fix, which preserves the spirit of #205. Thanks for the ticket and for the suggested fix!
